queries/powershell/highlights.scm

```diff
@@ -0,0 +1,324 @@
+    "Management.Automation.ProviderInfo" "System.Management.Automation.ProviderInfo"
+    "Management.Automation.PSDriveInfo" "System.Management.Automation.PSDriveInfo"
+    "Management.Automation.PSVariable" "System.Management.Automation.PSVariable"
+    "Management.Automation.AliasInfo" "System.Management.Automation.AliasInfo"
+    "Management.Automation.PathInfo" "System.Management.Automation.PathInfo"
+    "Management.Automation.PathInfoStack" "System.Management.Automation.PathInfoStack"
+    "Management.Automation.ApplicationInfo" "System.Management.Automation.ApplicationInfo"
+    "Management.Automation.CmdletInfo" "System.Management.Automation.CmdletInfo"
+    "Management.Automation.ExternalScriptInfo" "System.Management.Automation.ExternalScriptInfo"
+    "Management.Automation.FunctionInfo" "System.Management.Automation.FunctionInfo"
+    "Management.Automation.CommandTypes" "System.Management.CommandTypes"
+    "Management.Automation.ScopedItemOptions" "System.Management.Automation.ScopedItemOptions"
+    "Management.Automation.PSTypeName" "System.Management.Automation.PSTypeName"
+    "Management.Automation.ParameterMetadata" "System.Management.Automation.ParameterMetadata"
+    "Management.Automation.CommandParameterSetInfo"
+    "System.Management.Automation.CommandParameterSetInfo"
+    "Management.Automation.SessionStateEntryVisibility"
+    "System.Management.Automation.SessionStateEntryVisibility" "Management.Automation.FilterInfo"
+    "System.Management.Automation.FilterInfo" "Management.Automation.ModuleType"
+    "System.Management.Automation.ModuleType" "Management.Automation.PSModuleInfo"
+    "System.Management.Automation.PSModuleInfo" "Management.Automation.PSCustomObject"
+    "System.Management.Automation.PSCustomObject" "Management.Automation.PSScriptCmdlet"
+    "System.Management.Automation.PSScriptCmdlet" "Management.Automation.ErrorRecord"
+    "System.Management.Automation.ErrorRecord" "Management.Automation.PSCredential"
+    "System.Management.Automation.PSCredential" "Management.Automation.PSMethod"
+    "System.Management.Automation.PSMethod")
+  )
+
+; Function definitions
+;---------------------
+(function_statement
+  (function_name) @function)
+
+; Classes, fields and methods definitions
+;----------------------------------------
+(class_property_definition
+  (variable) @variable.member)
+
+(class_method_definition
+  (simple_name) @function.method)
+
+(class_statement
+  (simple_name) @_class_name
+  (class_method_definition
+    (simple_name) @constructor)
+  (#eq? @constructor @_class_name))
+
+; Functions, methods calls and properties access
+;--------------------------
+(command_invokation_operator) @operator
+
+(invokation_expression
+  (member_name) @function.call)
+
+(invokation_expression
+  (member_name) @constructor
+  (#eq? @constructor "new"))
+
+(member_access
+  (member_name
+    [
+      (simple_name)
+      (variable)
+    ] @variable.property))
```

Comment:
You are only allowed to use the captures listed in https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#highlights.

```suggestion
    ] @variable.member))
```